### ISSUE TITLE
[FEATURE] New basemap(s)

### ISSUE THREAD

**Issue by nilsnolde** [valhalla/web-app#127]
### What kind of feature you are suggesting?

Let's get Stadia's Alidade base map as the default (and keep OSM for the time being).

### Does it solve any problem?

OSM base map is not good for routing.

### Additional information

https://stadiamaps.com/themes/

Need to generate a API key. This is best done via Github Actions so we can encode the API key there. Needs a Github Action first though:)

**Comment by hs7898753** [comment]
Yes, Stadia's Alidade base map is a good option for routing. it provides more detailed and accurate data other than that it offers a variety of map themes and styles.

**Comment by nilsnolde** [comment]
Thanks, but this is an admin task. Gotta create an account, register an API key, set that on Github Secrets, get a SSH key pair for our planet server and register that as well in Secrets. The actual action is written in 10 mins.

**Comment by mustaphaturhan** [comment]
@nilsnolde do we still want this?

**Comment by nilsnolde** [comment]
Yeah I think so @mustaphaturhan. I can take care of that in a bit. We can ask them to raise the limits of the API key, they're also Valhalla users & contributors and generally nice people:)

**Comment by mustaphaturhan** [comment]
I am not sure if we really need a github action though. As far as I know, map api keys are easily "leakable" by default since everyone who knows how to use developer tools can analyze the url and get api key from parameter (the vector request being sent to 
https://tiles.stadiamaps.com/data/openmaptiles/12/2102/1347.pbf?api_key=your-api-key). We should use [domain based authentication](https://docs.stadiamaps.com/authentication/#domain-based-authentication).

So it should be fine to manually create an API key and add it to our `.env` file.

**Comment by nilsnolde** [comment]
Yeah, good point!
